src/zoo/Animal.java

```diff
@@ -5,6 +5,10 @@ public class Animal {
     private int age;
     private boolean isWild;
 
+    public boolean getIsWild() {
```

Comment:
in such getters, we tend to just name them `isWild` (omitting `get` prefix) :) this way the code reads `animal.isWild` as in `animal: is it wild? yes or no`. It's more English then :)

Reply:
ok, my mistake, done:)